client/src/components/SharedComponents/TopBar.jsx

```diff
@@ -5,25 +5,24 @@ import PropTypes from 'prop-types';
 import Logo from './TopBarComponents/Logo.jsx';
 import NotificationIcon from './TopBarComponents/NotificationIcon.jsx';
 
-class TopBar extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-
-    };
-  }
-
-  render () {
-    return (
-      <div id="topbar">
-        <div id="emptyDiv"/>
-        <Logo id="logo"/>
-        <NotificationIcon id="notificaitonIcon" />
-      </div>
-
-    )
-  }
+const TopBar = (props) => {
+  const { logoOnClick, notificationOnClick, newNotifications, loggedIn } = props;
+  return (
+    <div id="topbar">
+      <div id="emptyDiv" />
+      <Logo id="logo" onClick={logoOnClick} />
+      {
+        loggedIn ? (
+          <NotificationIcon id="notificaitonIcon"
```

Comment:
Super minor, you have a typo on the id for notificationIcon, not sure if that may cause confusion later on if you make edits.